services-api/src/main/java/io/scalecube/services/registry/api/ServiceRegistry.java

```diff
@@ -20,4 +21,6 @@
   boolean registerService(ServiceEndpoint serviceEndpoint);
 
   ServiceEndpoint unregisterService(String endpointId);
+
```

Comment:
Add `stop` or better `close()` . We need to have cleanup phase since we do have `listen()`  hence we have a subscribers layer hence we have to have a cleanup phase to tell subscribres bye bye.

Reply:
done